src/Services/Match/Match.Application/Mappers/ChatMapping.cs

```diff
@@ -0,0 +1,14 @@
+using Match.Application.DTOs.Chat.Request;
+using Match.Application.DTOs.Chat.Response;
+using Match.Domain.Models;
+
+namespace Match.Application.Mappers;
+
+public class ChatMapping : AutoMapper.Profile
+{
+    public ChatMapping()
+    {
+        CreateMap<CreateChatDto, Chat>();
+        CreateMap<Chat, ChatResponseDto>();
```

Comment:
It is better to explicitly specify which fields are mapped to which ones. Since if you do not specify this and further change the name of some field, there will be empty data and it will be difficult to find what the problem is